forge/db/controllers/ProjectTemplate.js

```diff
@@ -1,4 +1,5 @@
 const validSettings = [
+    'credentialSecret',
```

Comment:
This isn't needed - we aren't adding `credentialSecret` to the settings a Template can provide. It is being stored separately in ProjectSettings.